bitnami/sonarqube/values.yaml

```diff
@@ -900,38 +900,39 @@ postgresql:
     ## @param postgresql.auth.database Database to create when deploying the PostgreSQL chart
     ##
     database: bitnami_sonarqube
-  ## PostgreSQL service
+  ## @section PostgreSQL Primary parameters
```

Comment:
Please remove the `@section` of this line, it is part of the postgresql.

Reply:
Done